### PR TITLE
Artifact to install Chrome on a DevTest Lab VM

### PR DESCRIPTION
**Pull request by estebang**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@estebang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@estebang__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: Artifacts/windows-chrome/Artifactfile.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Chrome",
+  "description": "Installs Chrome using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Chrome"
```
Comment (drwill-ms) [inline Artifacts/windows-chrome/Artifactfile.json]: Let's add a Google tag as well.

**Comment by drwill-ms** [comment]
Looks good to me. Thanks for adding this artifact. I'll merge it after the update to tags.

Cheers!

File: Artifacts/windows-chrome/Artifactfile.json (hunk 1 above)
Comment (estebang) [inline Artifacts/windows-chrome/Artifactfile.json]: Done!

**Pull request merged**